testnets/doravotatestnet/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "doravotatestnet",
+    "assets": [
+      {
+        "description": "The native staking and governance token of the Theta testnet version of the Dora Vota.",
+        "denom_units": [
+          {
+            "denom": "uDORA",
+            "exponent": 0
+          },
+          {
+            "denom": "DORA",
+            "exponent": 6
+          }
+        ],
+        "base": "uDORA",
+        "name": "Dora Vota",
+        "display": "DORA",
+        "symbol": "DORA",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/cudostestnet/images/doravota.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/cudostestnet/images/doravota.svg"
```

Comment:
since these two images have a different appearance, I'd ask we only reference one of them to be the token logo under logo_URI's.
Although, the `images` property should allow for alternative logos.

Reply:
Yes, the name of our chain is `Dora Vota` (the testnet is `Dora Vota Testnet`), so that's what I'm setting up here is `doravotatestnet`.

Reply:
already update